src/resolver/strategies/generic/resolve.js

```diff
@@ -54,7 +55,10 @@ export default async function resolveGenericStrategy(options) {
       parameterMacro,
       modelPropertyMacro,
       useCircularStructures,
-      // eslint-disable-next-line camelcase
-    }).then(skipNormalization ? async (a) => a : normalize);
+    });
+
+    result.spec = skipNormalization ? result.spec : strategy.normalize(result.spec);
```

Comment:
```suggestion
    if (!skipNormalization) {
      result.spec = strategy.normalize(result.spec)
    }
```

Not sure which one is more readable